.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "daily"
```

Comment:
There is a conversation to be had around weekly vs. daily. IMO - daily is fine if the Auto-merging Github Action I mentioned in the issue is implemented and we trust the CI/CD enough.

Edit: Issue #702

Reply:
I think the auto-merge is something to be explored and it would make things a lot easier. However, we're in the middle of making a lot of changes to the test suite to speed it up so it's probably safer if we go with weekly + manual review for now.

If everything goes well, making it automated would be a great future addition and I'll add it to the dev road map for the year.